lib/readiness.js

```diff
@@ -55,17 +55,11 @@ bedrock.events.on('bedrock-server.readinessCheck', async () => {
       return false;
     }
     if(!ready) {
-      let names = '';
-      for(const name in dependencies) {
-        if(dependencies[name].ready === false) {
-          if(names === '') {
-            names = name;
-          } else {
-            names = `${names}, ${name}`;
-          }
-        }
-      }
-      logger.debug(`The readiness for ${names} failed. Retrying...`);
+      const names = Object.keys(dependencies)
+        .filter(n => !dependencies[n].ready);
+      logger.debug(
+        `The readiness for "${names.join(', ')}" failed. Retrying...`
+      );
```

Comment:
We don't drop trailing parenthesis -- I've seen this style leaking in in various places and it should be fixed wherever you see it (if you're working on that part of the code). Also, we should include the dependency names in the JSON-generated data so it can be parsed and used by other tools:

```suggestion
      logger.debug(
        `The readiness for "${names.join(', ')}" failed. Retrying...`,
        {dependencies: names});
```